IDEAS/Electric/Distribution/BaseClasses/Branch.mo

```diff
@@ -12,7 +12,7 @@ model Branch
   Modelica.Thermal.HeatTransfer.Interfaces.HeatPort_a port_a annotation (
       Placement(transformation(extent={{10,10},{30,30}}), iconTransformation(
           extent={{10,10},{30,30}})));
-  Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow prescribedHeatFlow
+  Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow prescribedHeatFlow(alpha=1)
```

Comment:
@arnoutaertgeerts the reason why you needed to add this is the following:
when the heatport is unconnected externally this forces Q_flow = 0. However Q_flow = Plos so this results into a singular system. Setting alpha =/= 0 allows to solve this because the port temperature will increase when the power cannot be delivered. This is however not physical.
I propose to make the calculation of Plos conditional: add a boolean parameter to enable the heatport/prescribedheatflow and the calculation of Plos. When enabling this parameter port_a must be connected.